src/Sulu/Bundle/MediaBundle/Twig/MediaTwigExtension.php

```diff
@@ -50,15 +51,23 @@ public function getFunctions()
      * @param int|MediaInterface $media id to resolve
      * @param string $locale
      *
-     * @return MediaApi
+     * @return MediaApi|null
      */
     public function resolveMediaFunction($media, $locale)
     {
+        if (!$media) {
+            return;
+        }
+
         if (is_object($media)) {
             return $this->resolveMediaObject($media, $locale);
         }
 
-        return $this->mediaManager->getById($media, $locale);
+        try {
+            return $this->mediaManager->getById($media, $locale);
+        } catch (MediaNotFoundException $e) {
```

Comment:
no multiple medias use the getByIds function which does not throw an exception when a media not exist. the not existing media is just ignored.